docs/UserGuide.md

```diff
@@ -163,6 +163,26 @@ Expected outputs when the command succeeds:
 Expected outputs when command fails:
 * `Unable to edit the patient: Patient identification does not exist.`
 
+### Undoing a command: `undo`
+
+Undoes an undoable command within the address book.
```

Comment:
I think this part is well-written! However, I'm concerned that there could be some ambiguity regarding this, because readers may think it's a command that can't be done instead. Maybe it can be replaced with 'undo-able' for clarity?

Reply:
Yes, I agree that 'undo-able` would provide better clarity.